galasa-parent/dev.galasa.framework.api.authentication/src/main/java/dev/galasa/framework/api/authentication/Authorization.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package dev.galasa.framework.api.authentication;
+
+import java.util.StringTokenizer;
+
+import javax.servlet.http.HttpServletRequest;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.interfaces.DecodedJWT;
+
+public class Authorization {
+
+    protected DecodedJWT decodedJwt;
+    private  String jwt;
+
+    public Authorization (HttpServletRequest req) {
+        String galasaHeader = req.getHeader("Galasa-Application");
+        if (galasaHeader != null){
```

Comment:
Also, I think we can remove this check since it won't matter whether or not the "Galasa-Application" header exists.